github4s/src/main/scala/github4s/algebras/Projects.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2016-2020 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package github4s.algebras
+
+import github4s.GithubResponses.GHResponse
+import github4s.domain._
+
+trait Projects[F[_]] {
+
+  /**
+   * List the projects belonging to a specific organization
+   *
+   * @param org        Organization for which we want to retrieve the members
+   * @param state      Filter projects returned by their state. Can be either `open`, `closed`, `all`.
+   *                   Default: `open`
+   * @param pagination Limit and Offset for pagination
+   * @param headers Optional user headers to include in the request
+   * @return GHResponse with the list of projects belonging to this organization
+   */
+  def listProjects(
+      org: String,
+      state: Option[String] = None,
+      pagination: Option[Pagination] = None,
+      headers: Map[String, String] = Map()
+  ): F[GHResponse[List[Project]]]
+
+  /**
+   * List the columns belonging to a specific project id
+   *
+   * @param project_id Project id for which we want to retrieve the columns
+   * @param pagination Limit and Offset for pagination
+   * @param headers Optional user headers to include in the request
+   * @return GHResponse with the list of columns belonging to this project id
+   */
+  def listColumns(
+      project_id: Int,
```

Comment:
```suggestion
      projectId: Int,
```
wdyt?

Reply:
In my opinion, I think we should leave it at that and to have the same structure in the whole project.
For instance:
[client_id](https://github.com/47deg/github4s/blob/master/github4s/src/main/scala/github4s/domain/Authorization.scala#L28)
[commit_id](https://github.com/47deg/github4s/blob/master/github4s/src/main/scala/github4s/domain/PullRequest.scala#L113)